modules/statistics/src/main/frontend/src/Statistics/Statistic.jsx

```diff
@@ -221,16 +255,23 @@ function Statistic(props) {
               <Label value={definition["x-label"]} offset={-10} position="insideBottom" />
             </XAxis>
             <YAxis allowDecimals={false} label={{ value: definition["y-label"], angle: -90, position: 'insideLeft', offset: 10 }} />
-            <Tooltip />
+            <Tooltip content={<CustomTooltip />} />
             {isSplit && <Legend align="right" verticalAlign="top" height={legendHeight} />}
             {allFields.map((field, idx) =>
               isBar ?
-                <Bar dataKey={field.label || field} fill={chartColours[idx]} key={idx} onClick={(data, index) => handleClick(data, field)} style={customStyle}/>
+                <Bar
+                  dataKey={field.label || field}
+                  stackId="a"
```

Comment:
They should not be stacked unless we're grouping null and false for xvar
```suggestion
                  stackId={groupNullAndFalseAnswersForXVar ? "a" : ""}
```